server/controllers/tournamentController.ts

```diff
@@ -21,12 +21,12 @@ const getAllTournaments = catchAsync(async (req, res) => {
   const pageSize = Number(req.query.pageSize) || 100;
 
   const data = await TournamentRepository.getAll(page, pageSize);
-  const enrichedTournament = await Tournament.enrichTournamentsWithMSUsers(data.tournaments, `Bearer ${req.cookies.jwt}`);
+  const enrichedTournaments : Tournament[] = await Tournament.enrichTournamentsWithMSUsers(data.tournaments, `Bearer ${req.cookies.jwt}`);
```

Comment:
Is this type required?
```suggestion
  const enrichedTournaments = await Tournament.enrichTournamentsWithMSUsers(data.tournaments, `Bearer ${req.cookies.jwt}`);
```

Reply:
no , deleted